data/mods/MindOverMatter/effectoncondition/eoc_mental_engineering.json

```diff
@@ -0,0 +1,4834 @@
+              }
+            ]
+          },
+          {
+            "case": 100,
+            "effect": [
+              {
+                "run_eocs": [
+                  {
+                    "id": "EOC_TELEPATH_MENTAL_ENGINEERING_SIDE_EFFECTS_STYLISH_2",
+                    "effect": [
+                      { "u_lose_trait": "STYLISH" },
+                      {
+                        "u_message": "Your previous source of clothing-based happiness means nothing to you.",
+                        "type": "bad"
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "case": 101,
+            "effect": [
+              {
+                "run_eocs": [
+                  {
+                    "id": "EOC_TELEPATH_MENTAL_ENGINEERING_SIDE_EFFECTS_MASOCHIST_1",
+                    "effect": [
+                      { "u_mutate_towards": "MASOCHIST" },
+                      { "u_message": "Even pinching your skin feels better than it used to.", "type": "good" }
+                    ]
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "case": 102,
+            "effect": [
+              {
+                "run_eocs": [
+                  {
+                    "id": "EOC_TELEPATH_MENTAL_ENGINEERING_SIDE_EFFECTS_MASOCHIST_2",
+                    "effect": [
+                      { "u_lose_trait": "MASOCHIST" },
+                      {
+                        "u_message": "The aches and pains of combat suddenly seem far less appealing.",
+                        "type": "bad"
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "case": 103,
+            "effect": [
+              {
+                "run_eocs": [
+                  {
+                    "id": "EOC_TELEPATH_MENTAL_ENGINEERING_SIDE_EFFECTS_SPIRITUAL_1",
+                    "effect": [
+                      { "u_mutate_towards": "SPIRITUAL" },
+                      {
+                        "u_message": "You can now see the things unexplainable by science.  And right now, that's your entire life.",
+                        "type": "good"
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "case": 104,
+            "effect": [
+              {
+                "run_eocs": [
+                  {
+                    "id": "EOC_TELEPATH_MENTAL_ENGINEERING_SIDE_EFFECTS_SPIRITUAL_2",
+                    "effect": [
+                      { "u_lose_trait": "SPIRITUAL" },
+                      { "u_message": "Your prior source of comfort with your life is now gone.", "type": "bad" }
+                    ]
+                  }
+                ]
+              }
+            ]
+          },
```

Comment:
is there a reason all this 108 switch-cases should run custom eocs

```json
          {
            "case": 104,
            "effect": [
              {
                "run_eocs": [
                  {
                    "id": "EOC_TELEPATH_MENTAL_ENGINEERING_SIDE_EFFECTS_SPIRITUAL_2",
                    "effect": [
                      { "u_lose_trait": "SPIRITUAL" },
                      { "u_message": "Your prior source of comfort with your life is now gone.", "type": "bad" }
                    ]
                  }
                ]
              }
            ]
          },
```


and not just, yknow

```json
  {
    "case": 104,
    "effect": [
      { "u_lose_trait": "SPIRITUAL" },
      { "u_message": "Your prior source of comfort with your life is now gone.", "type": "bad" }
    ]
  },
```

Reply:
That was an artifact of when those used to be full EOC's with conditions.  I removed the conditions and swapped over to u_mutate_towards, but I see now that I can also just remove the sub-eoc's entirely.  I just submitted the push and it looks like it took out a good chunk of it.